sonar-project.properties

```diff
@@ -14,7 +14,7 @@ sonar.cfamily.build-wrapper-output=build-wrapper-output-dir
 sonar.cfamily.gcov.reportsPath=.
 sonar.coverageReportPaths=cov.xml
 sonar.python.coverage.reportPaths=jormungandr_serpy_integration_coverage.xml,jormungandr_coverage_serpy.xml,tyr_integration_coverage.xml
-sonar.coverage.exclusions=source/jormungandr/tests/**,source/tyr/tests/**,source/benchmark/**,source/cities/**,source/eitri/**,source/log_analyzer/**,source/monitor/**,source/scripts/**,source/sql/**,source/third_party/**,source/tools/**
+sonar.coverage.exclusions=source/jormungandr/tests/**,source/tyr/tests/**,source/benchmark/**,source/cities/**,source/eitri/**,source/log_analyzer/**,source/monitor/**,source/scripts/**,source/sql/**,source/third_party/**,source/tools/**,source/tests/**
```

Comment:
```suggestion
sonar.coverage.exclusions=source/**/tests/**,source/benchmark/**,source/cities/**,source/eitri/**,source/log_analyzer/**,source/monitor/**,source/scripts/**,source/sql/**,source/third_party/**,source/tools/**
```

Reply:
are you sure this syntax is allowed ? I

Reply:
It looked like standard bash glob so I assumed it was but you might be right, I may have extrapolated :shrug: 

That said, it seems [to be possible](https://docs.sonarqube.org/latest/analysis/test-coverage/test-coverage-parameters/).

Reply:
allright, let's try it !